gov/governance/_GET_proposal.gno

```diff
@@ -21,19 +21,19 @@ func GetProposalTypeByProposalId(proposalId uint64) string {
 	if !exist {
 		panic(addDetailToError(
 			errDataNotFound,
-			ufmt.Sprintf("_GET_proposal.gno__GetProposalTypeByProposalId() || proposalId(%d) not found", proposalId),
+			ufmt.Sprintf("proposalId(%d) not found", proposalId),
 		))
 	}
 
-	return proposal.ProposalType
+	return proposal.ProposalType.String()
 }
 
 func GetYeaByProposalId(proposalId uint64) string {
 	proposal, exist := proposals[proposalId]
 	if !exist {
 		panic(addDetailToError(
 			errDataNotFound,
-			ufmt.Sprintf("_GET_proposal.gno__GetYeaByProposalId() || proposalId(%d) not found", proposalId),
+			ufmt.Sprintf("proposalId(%d) not found", proposalId),
```

Comment:
`!exist ~ panic` are being used over and over.
make `mustGet` func and raise panic from it.

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/448/commits/4af3dee33556940a149dccd4daca5faf07eded46#diff-d5b8c62fc4099f1536ff43a2ca412e7e2ab2d3e3777e3cc63fe691436dfb9d2e